saleor/settings_demo.py

```diff
@@ -18,8 +18,8 @@
 
 USE_JSON_CONTENT = True
 
-PWA_ORIGIN = get_list(os.environ.get("PWA_ORIGIN", "pwa.saleor.io"))
-PWA_DASHBOARD_URL_RE = re.compile("^https?://%s/dashboard/.*" % PWA_ORIGIN)
+PWA_ORIGINS = get_list(os.environ.get("PWA_ORIGIN", "pwa.saleor.io"))
```

Comment:
For consistency we could also rename the env variable:
```suggestion
PWA_ORIGINS = get_list(os.environ.get("PWA_ORIGINS", "pwa.saleor.io"))
```